about/overview-openshift-serverless-logic.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="overview-openshift-serverless-logic"]
+= Openshift Serverless Logic overview
+:context: overview-openshift-serverless-logic
+
+
+toc::[]
+
+OpenShift Serverless Logic enables developers or architects to define declarative workflow models that orchestrate event-driven, serverless applications.
```

Comment:
```suggestion
OpenShift Serverless Logic enables developers or architects to create cloud-native applications using declarative workflow models that orchestrate event-driven, serverless applications.
```

nitpick, kind of missing the notion of cloudnativeapps, with my suggestion it feels too long thought so I am not sure.

Reply:
I think that the original text here is good, because it introduces the main concept of "workflows" to do, bla, bla.